src/main/java/org/example/validators/DeliveryEmployeeValidator.java

```diff
@@ -0,0 +1,40 @@
+package org.example.validators;
+
+import org.example.exceptions.Entity;
+import org.example.exceptions.InvalidException;
+import org.example.models.DeliveryEmployeeRequest;
+
+public class DeliveryEmployeeValidator {
+
+    public void validateDeliveryEmployee(
+            final DeliveryEmployeeRequest deliveryEmployeeRequest)
+            throws InvalidException {
+        final int maxNameLength = 50;
+        final double maxSalary = 9999999999.99;
```

Comment:
How I would do it is (don't need to fix): final double maxSalary = 10000000 and then compare using less than

Also, I think it would be better with <= rather than > as it's easier to read